### PR TITLE
Feature/uri form serialization

### PR DESCRIPTION
**Pull request by samwelkanda**
Related to https://github.com/microsoft/kiota/issues/2075

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1 @@
+* @samwelkanda @shemogumbe @baywet @darrelmiller @ddyett @andrueastman @silaskenneth @ndiritu
```
Comment (baywet) [inline .github/CODEOWNERS]:
```suggestion
* @microsoft/kiota-write
```

let's start to align all the repositories

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -0,0 +1,42 @@
+name: Python Serialization Form
+
+on:
+  workflow_dispatch:
+  workflow_call:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main, dev]
```
Comment (baywet) [inline .github/workflows/build.yml]: do we plan to have a dev branch at this point?

File: .github/workflows/publish.yml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+          python-version: 3.12
+      - name: Install flit
+        run: |
+          pip install flit
+      - name: Publish the distibution to PyPI
+        run: flit publish
+        env:
+          FLIT_INDEX_URL: https://upload.pypi.org/legacy/
+          FLIT_USERNAME: __token__
+          FLIT_PASSWORD: ${{ secrets. PYPI_API_TOKEN }}
```
Comment (baywet) [inline .github/workflows/publish.yml:35]: do you need me to set up the token? please reach out on teams

File: kiota_serialization_form/form_parse_node.py [hunk 4]
```diff
@@ -0,0 +1,366 @@
...
+        #     keys = camel_case_key.split(',')
+        #     if len(keys) > 1:
+        #         result = []
+        #         for key in keys:
+        #             key_name = key[0].upper() + key[1:]
+        #             if key_name in enum_class.__members__:
+        #                 result.append(enum_class[key_name])
+        #         return result
+        #     raise Exception(f'Invalid key: {camel_case_key} for enum {enum_class}.')
+        # return None
```
Comment (baywet) [inline kiota_serialization_form/form_parse_node.py]: commented out code

File: kiota_serialization_form/form_serialization_writer.py [hunk 5]
```diff
@@ -0,0 +1,356 @@
...
+            #         method = getattr(self, f'write_{value_type.__name__.lower()}_value')
+            #         method(None, value)
+            #     elif isinstance(value, Parsable):
+            #         self.write_object_value(None, value)
+            #     elif hasattr(value, '__dict__'):
+            #         self.write_non_parsable_object_value(None, value)
+            #     else:
+            #         raise TypeError(
+            #             f"Encountered an unknown type during serialization {value_type}"
+            #         )
```
Comment (baywet) [inline kiota_serialization_form/form_serialization_writer.py]: commented out code

**Review by baywet — CHANGES_REQUESTED** [review]

File: kiota_serialization_form/form_parse_node.py (hunk 4 above)
Comment (samwelkanda) [reply kiota_serialization_form/form_parse_node.py]: Removed

**Review by baywet — APPROVED** [review]
thanks for making the changes!

**Review by baywet — APPROVED** [review]

**Review by baywet — APPROVED** [review]

**Pull request merged**